validator/flags/flags.go

```diff
@@ -23,6 +23,17 @@ var (
 		Name:  "tls-cert",
 		Usage: "Certificate for secure gRPC. Pass this and the tls-key flag in order to use gRPC securely.",
 	}
+	// SlasherRPCProviderFlag defines a slasher node RPC endpoint.
+	SlasherRPCProviderFlag = &cli.StringFlag{
+		Name:  "slasher-rpc-provider",
+		Usage: "Slasher node RPC provider endpoint",
+		Value: "localhost:4000",
```

Comment:
The default is the same address as the beacon rpc?

Reply:
Missed it, thanks

Reply:
changed to 4001